ledfx/sendspin/stream.py

```
@@ -110,67 +120,105 @@ def _audio_chunk_handler(
+
+        Extracted so both the PCM path and the callback-driven FLAC backend
+        (Stage 2) share identical leftover-handling and scheduling logic
+        without duplication.  Keeping the scheduling in one place also means
+        the FLAC backend can be purely callback-driven (pyFLAC fires this
+        from within process()) without any duplicate heap/leftover state.
+
+        Args:
+            samples:      Mono float32 numpy array of decoded audio.
+            play_time_us: Intended play time (µs) for the first sample in
+                          *samples*.  Ignored when leftover samples from the
+                          previous chunk are prepended – in that case the
```

Ruff (0.15.9)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)